lib/operations/collection_ops.js

```diff
@@ -68,8 +186,8 @@ function bulkWrite(coll, operations, options, callback) {
   // Create the bulk operation
   const bulk =
     options.ordered === true
-      ? coll.initializeOrderedBulkOp(options)
-      : coll.initializeUnorderedBulkOp(options);
+      ? ordered(coll.s.topology, coll, options)
```

Comment:
I realize this isn't the first time we have called them this, but it occurs to me now that importing these functions as `unordered`/`ordered` is _way_ too generic. Can we call the operations `initializeOrderedOp(eration)` and the same for unordered?